### PR TITLE
script and test for T0

### PR DESCRIPTION
**Pull request by vergep**

### CODE REVIEW

File: lir/classifiers.py [hunk 1]
```diff
@@ -34,3 +36,53 @@ def TLM_calc_means(X, y):
...
+    grouped = df.groupby(axis='index', by='label')
+
+    # calculate kappa
+    # get the number of repetitions per source
+    reps = np.array(grouped.size()).reshape((-1, 1))
+    # calculate the sum of the repetitions squared and kappa;
+    #   kappa represents the average number of repetitions per source
+    sum_reps_sq = sum(reps**2)
+    n_sources = len(reps)
+    kappa = float((reps.sum() - sum_reps_sq/reps.sum())/(n_sources-1))
```
Comment (camillevandijk) [inline lir/classifiers.py:76]: This yields a warning in the test environment: DeprecationWarning: Conversion of an array with ndim > 0 to a scalar is deprecated, and will error in future. Ensure you extract a single element from your array before performing this operation. (Deprecated NumPy 1.25.)
  kappa = float((reps.sum() - sum_reps_sq/reps.sum())/(n_sources-1))

I found we can solve this by using (reps.sum() - sum_reps_sq/reps.sum())/(n_sources-1)).item() instead.
See https://stackoverflow.com/questions/30311172/convert-list-or-numpy-array-of-single-element-to-float-in-python

File: lir/classifiers.py [hunk 2]
```diff
@@ -34,3 +36,53 @@ def TLM_calc_means(X, y):
     means = grouped.mean()
     return np.array(means)
 
+def TLM_calc_h_sq(X, y):
+    """
+    X np.array of measurements, rows are objects, columns are variables
```
Comment (kimdebie) [inline lir/classifiers.py:41]: again - docstring and more explicit function name. What is h and what do we need it for? Also include an explanation of the formula for h in the docstring.

File: tests/data/TLM/R_output/MSwithin.csv [hunk 3]
```diff
@@ -0,0 +1,11 @@
+"K39","Ti49","Mn55","Rb85","Sr88","Zr90","Ba137","La139","Ce140","Pb208"
```
Comment (kimdebie) [inline tests/data/TLM/R_output/MSwithin.csv:1]: folder/file names:

- data: rename to `resources` 
- TLM: `two_level_model`
- R_output: `output`
- XDATA/YDATA/ZDATA: any way to give this more explicit names?

**Comment by vergep** [comment]
PR closed since it is covered by PR #91

**Pull request closed without merging**